examples/inline.rs

```diff
@@ -28,10 +28,9 @@ use ratatui::{
     layout::{Alignment, Constraint, Layout, Rect},
     style::{Color, Modifier, Style},
     symbols,
-    terminal::{Frame, Terminal, Viewport},
     text::{Line, Span},
     widgets::{block, Block, Gauge, LineGauge, List, ListItem, Paragraph, Widget},
-    TerminalOptions,
+    Frame, Terminal, TerminalOptions, Viewport,
```

Comment:
I like this change. It's already mixed up in the examples (which should be relatively clean in the first place) so it will be even more cleanup in the real world. :sparkles:

Reply:
2e400f892